plugins/build.gradle.kts

```diff
@@ -66,11 +67,18 @@ dependencies {
   implementation("com.google.code.gson:gson:2.8.9")
   implementation(libs.android.gradlePlugin.gradle)
   implementation(libs.android.gradlePlugin.builder.test.api)
+  implementation("io.github.pdvrieze.xmlutil:serialization-jvm:0.90.3") {
```

Comment:
How important is this dependency to get the functionality we need? We need to be conservative when adding dependencies, specially if they require different maven repos

Reply:
I hear you. Let me know if you think the merits outweigh the con of bringing it in:

- Saves us approximately ~500 lines of [de/se]rialization code.
- Avoids us needing to test serialization in isolation.
- [Recommended for XML usage from the `kotlinx.serialization` folks](https://github.com/Kotlin/kotlinx.serialization/issues/1747#issuecomment-956070074).
- Has pretty descriptive errors when deserialization failures occur.

I opted for it to avoid adding additional overhead as far as maintenance is concerned, as it's a decent amount of code to properly handle pom files. But, if you think it's still not worth bringing in as a dependency- and you'd rather we do it in-house- then I can add code to do that instead. wdyt?

Reply:
Fixed:)